src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_commons.py

```diff
@@ -153,10 +153,10 @@ def assert_output_path_writable(output_path: str,
 
 def write_ndarray_to_tsv(output_file: str,
                          array: np.ndarray,
-                         comment=io_consts.default_comment_char,
+                         comment_char=io_consts.default_comment_char,
```

Comment:
I think the `comment` parameter name was fine (since this is what is used by pandas, etc.), but up to you.

Reply:
Done